src/ssl.c

```diff
@@ -48611,9 +48611,8 @@ int wolfSSL_RSA_private_decrypt(int len, const unsigned char* fr,
     return ret;
 }
 
-#if !defined(_WIN32) && !defined(HAVE_SELFTEST) && \
-    (!defined(HAVE_FIPS) || \
-        (defined(HAVE_FIPS_VERSION) && HAVE_FIPS_VERSION > 2))
+#if !defined(HAVE_SELFTEST) && (!defined(HAVE_FIPS) || \
```

Comment:
@haydenroche5  Can you also update the comment on the `#endif`? It still has `_WIN32`.